src/bots/google/Gemini20FlashAPIBot.js

```
@@ -0,0 +1,11 @@
+import GeminiAPIBot from "./GeminiAPIBot";
+
+export default class Gemini20FlashAPIBot extends GeminiAPIBot {
+  static _className = "Gemini20FlashAPIBot";
+  static _logoFilename = "gemini-2.0-flash-logo.png";
+  static _model = "gemini-2.0-flash-001";
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)